commands/filemanager.ts

```diff
@@ -14,10 +9,6 @@ exports.command = 'filemanager';
 exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.builder = yargs => {
-  addOverwriteOptions(yargs);
-  addConfigOptions(yargs);
-  addAccountOptions(yargs);
```

Comment:
This is something that maybe we should find a consistent pattern for. Seems like there are a lot of redundant `addOptions` calls between the actual command files and the files for the command bucket. Do you think it makes sense to just add them individually? In this case it looks like there are a few shared ones between filemanager fetch and upload that we could add in the bucket.

I also took advantage of the buckets to not have to add `addGlobalOptions` to every command individually

Reply:
I don't have a very strong opinion on this, but my reasoning for being redundant and moving all of these options to the commands is because it's easy to forget what's being declared at this level. We had an issue where we were calling `addAccountOptions(yargs)` in these command bucket files and also in the individual commands themselves.

I think ideally we just use middleware for stuff like this (where we need it to apply to every command), but in the meantime it feels slightly more straightforward to me to keep the definitions in each file just so we can quickly understand the full set of options that each command is subscribing to. But that's just my reasoning for why I did it this way. I could be convinced either way tbh.

Reply:
> I think ideally we just use middleware for stuff like this (where we need it to apply to every command), but in the meantime it feels slightly more straightforward to me to keep the definitions in each file just so we can quickly understand the full set of options that each command is subscribing to. But that's just my reasoning for why I did it this way. I could be convinced either way tbh.

+1 on this.  I do like the ease of adding them flags in the buckets, but putting the flags in the final commands is so much easier to reason about.